systers_portal/meetup/models.py

```diff
@@ -42,9 +42,13 @@ def __str__(self):
         return self.title
 
     def clean_fields(self, *args, **kwargs):
-        """Validate meetup date is not less than today's date"""
+        """Validate meetup date is not less than today's date and meetup time is not a time
+        that has already passed."""
         if self.date < timezone.now().date():
             raise ValidationError("Date should not be less than today's date.")
+        if self.time:
```

Comment:
I read about the model and form validation. It does seem better to use form validation. I made the changes.